api/src/test/java/marquez/common/UtilsTest.java

```diff
@@ -98,6 +104,77 @@ public void testChecksumFor_notEqual() {
     assertThat(checksum0).isNotEqualTo(checksum1);
   }
 
+  @Test
+  public void testToUuid() {
+    final String uuidString = "156c934a-6f35-440b-ac88-7a0b6e83fe51";
+    final UUID uuid = Utils.toUuid(uuidString);
+    assertThat(uuid.toString()).isEqualTo(uuidString);
+  }
+
+  @Test
+  public void testToUuid_throwsOnBlankOrEmpty() {
+    assertThatExceptionOfType(IllegalArgumentException.class).isThrownBy(() -> Utils.toUuid(""));
+    assertThatExceptionOfType(IllegalArgumentException.class).isThrownBy(() -> Utils.toUuid(" "));
+  }
+
+  @Test
+  public void testToUuid_throwsOnLengthNotValid() {
+    assertThatExceptionOfType(IllegalArgumentException.class)
+        .isThrownBy(() -> Utils.toUuid("53678291-e1fa-4f1b-bb29-939b-53a855c48817"));
+    assertThatExceptionOfType(IllegalArgumentException.class)
+        .isThrownBy(() -> Utils.toUuid("53678291-e1fa-4f1b-53a855c48817"));
+  }
+
+  @Test
+  public void testToInstantOrNull() {
+    final Instant nullInstant = Utils.toInstantOrNull(null);
+    assertThat(nullInstant).isNull();
+
+    final String asIso = "2021-01-09T19:49:24.201Z";
+    final Instant instant = Utils.toInstantOrNull(asIso);
```

Comment:
Jackson should be handling Instant behavior. Invalid requests should fail at the resource layer so I don't see the need for this.